src/app/(proper_react)/(redesign)/(authenticated)/user/welcome/EnterInfo.tsx

```diff
@@ -404,6 +413,12 @@ export const EnterInfo = ({
           )}
           <Button
             {...confirmDialogTrigger.triggerProps}
+            // react-aria sets `aria-expanded`, which causes this to be
+            // announced as "collapsed", rather than opening a dialog. By
+            // setting aria-haspopup, this interaction is clearer. See also:
+            // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup
+            aria-haspopup="dialog"
+            aria-expanded={undefined}
```

Comment:
Feeding back [from Jira](https://mozilla-hub.atlassian.net/browse/MNTOR-2765?focusedCommentId=859950) in case someone (probably me :sweat_smile: ) looks at this PR in the future: the problem was that when the form had errors, no dialog would pop up when clicking the button, which makes the "collapsed" announcement confusing.

Another solution would have been to simply disable the button until there were no errors in the form.